frontend/test/components/documentUnit/documentUnitList.spec.ts

```diff
@@ -111,6 +111,8 @@ describe("documentUnit list", () => {
           hasNote: true,
           isDeletable: false,
           isEditable: false,
+          source: "should not show source",
```

Comment:
Diese beiden Zeilen werden nicht abgetestet auf "Nicht-Sichtbarkeit". Würde entweder auf getText not.toBeVisible testen oder hier rausnehmen

Reply:
Ah, good catch. Das hatte ich vor zu testen, aber vergessen. Danke!